app/views/articles/_single_sponsor.html.erb

```diff
@@ -4,5 +4,5 @@
     <img src="<%= cloudinary(organization.nav_image_url) %>" alt="<%= organization.name %> logo" loading="lazy" class="partner-image-light-mode" />
     <img src="<%= cloudinary(organization.dark_nav_image_url || organization.nav_image_url) %>" alt="<%= organization.name %> logo" loading="lazy" class="partner-image-dark-mode" />
   </a>
-  <p><%= sponsorship.sponsorable_type == "ActsAsTaggableOn::Tag" ? sponsorship.blurb_html.html_safe : sponsorship.tagline %></p>
-</div>
\ No newline at end of file
+  <p><%= sponsorship.sponsorable_type == "ActsAsTaggableOn::Tag" ? sanitize(sponsorship.blurb_html) : sponsorship.tagline %></p>
```

Comment:
Just wondering if there was no sanitization here originally to make the sponsorship blurb more flexible? cc: @benhalpern